### PR TITLE
DOCS-1456: Minor improvements in the NodeLocal DNS Cache topic

### PR DESCRIPTION
**Pull request by girishshankaran**
Minor enhancements in the topic.

Product Version(s):
 Calico Enterprise 3.17

Issue:
(https://tigera.atlassian.net/browse/DOCS-1456)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@bmckercher123 Can you review this one?

File: calico-enterprise/networking/configuring/node-local-dns-cache.mdx [hunk 1]
```diff
@@ -2,22 +2,17 @@
 description: Install NodeLocal DNSCache
 ---
 
-# Install NodeLocal DNSCache
+# Use NodeLocal DNSCache in your cluster
 
 ## Big picture
```
Comment (bmckercher123) [inline calico-enterprise/networking/configuring/node-local-dns-cache.mdx:7]: Suggest the following:

## Big picture
Set up NodeLocal DNSCache to improve DNS lookup latency. 

## Before you begin

**Required**
Follow these [steps](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to enable NodeLocal DNSCache connectivity.

**Unsupported**
- OpenShift
    NodeLocal DNSCache does not support OpenShift. Openshift has its own DNS caching mechanism which deploys CoreDNS pods as DaemonSets.
For more information, see [Openshift DNS setup](https://docs.openshift.com/container-platform/4.10/networking/dns-operator.html).

File: calico-enterprise/networking/configuring/node-local-dns-cache.mdx [hunk 2]
```diff
@@ -36,8 +31,8 @@ deployed to your cluster
 ## How to
```
Comment (bmckercher123) [inline calico-enterprise/networking/configuring/node-local-dns-cache.mdx]: ### Create a policy to allow traffic from NodeLocal DNSCache

The following is a sample network policy that allows all incoming TCP traffic....

**Review by bmckercher123 — APPROVED** [review]
Great improvements. A few suggestions at your discretion.

**Pull request merged**